src/plugins/upf/upf.c

```diff
@@ -214,6 +216,13 @@ vnet_upf_tdf_ul_enable_disable (fib_protocol_t fproto, u32 sw_if_index,
       upf_tdf_ul_lookup_add_i (vec_elt (gtm->tdf_ul_table[fproto],
 					fib_index), &pfx, fib_index);
 
+      /* Set IP46 enablement for SW interface with given index */
+      vnet_main_t *vnm = gtm->vnet_main;
+      vnet_interface_main_t *im = &vnm->interface_main;
+      u32 thread_index = vlib_get_thread_index ();
+      vlib_set_simple_counter (&im->sw_if_counters[FIB_PROTOCOL_IP4 == fproto ? 10 : 11],
```

Comment:
where does the magic `10` and `11` come from?

Reply:
I will include extra headers here with definition to make it more readable. However, there are places in vpp where constants are used instead of definitions.